expense-splitter/src/components/expense/ExpenseList.jsx

```diff
@@ -28,10 +28,8 @@ export default function ExpenseList() {
         <div className="content-end">{expense.amount}</div>
         <div className="flex content-end gap-2">
```

Comment:
Adding a justify-content: center to this div should align the iconbuttons with the rest of the text

Reply:
I tried that, it didn't work for some reason. I tried text align, align items. I can't seem to center the icons.

Reply:
This is going to change once we implement the Card component so it should be fine for now.